src/ddmd/globals.d

```diff
@@ -172,6 +172,10 @@ struct Param
     const(char)* moduleDepsFile;        // filename for deps output
     OutBuffer* moduleDeps;              // contents to be written to deps file
 
+    const(char)* makeDepsFile;          // filename for make-style dependencies output
+    const(char)* makeDepsTarget;        // the generated make target name
+    Array!(const(char)*)* makeDeps;     // array of paths that the compiled module(s) depend on
```

Comment:
I wonder if these are strictly needed, or we should start requesting that new driver-only fields be added privately in mars.d so as to not affect other backends.